### PR TITLE
Added test on sdk.locales static methods values

### PR DESCRIPTION
**Pull request by MarynaG**
- Added test, which verifies, that sdk.locales static methods have expected values

- Refactored open-sdk-locales-data-test.js.

### CODE REVIEW

File: test/cypress/integration/reusable/open-sdk-locales-data-test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const localesData = require('../fixtures/locales-data.json')
+
+export function verifySdkLocalesData(iframeSelector) {
+  cy.getSdk(iframeSelector).then(sdk => {
+    expect(JSON.stringify(sdk.locales)).to.equal(JSON.stringify(localesData))
```
Comment (suevalov) [inline test/cypress/integration/reusable/open-sdk-locales-data-test.js]: `JSON.stringify` makes it hard to understand the mismatch between the expected and actual if there's an error.

Have you tried using `deep.equal`?

Comment (MarynaG) [reply test/cypress/integration/reusable/open-sdk-locales-data-test.js]: Thank you Alex, fixed the tests and they work.

File: test/cypress/integration/reusable/open-sdk-user-data-test.js [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-    expect(sdk.user.spaceMembership.admin).to.equal(user.isAdmin)
-    expect(sdk.user.spaceMembership.sys.id).to.equal(user.membershipId)
-    expect(sdk.user.sys.id).to.equal(user.userId)
+    expect(sdk.user).to.deep.equal(userData)
   })
 }
 
 export function openSdkUserDataTest(iframeSelector) {
-  it('sdk.user static methods have expected values', () => {
+  it.only('sdk.user static methods have expected values', () => {
```
Comment (suevalov) [inline test/cypress/integration/reusable/open-sdk-user-data-test.js]: 😨

Comment (davidfateh) [reply test/cypress/integration/reusable/open-sdk-user-data-test.js]: This should be a lint rule, no?

Comment (suevalov) [reply test/cypress/integration/reusable/open-sdk-user-data-test.js]: it definitely should!

Comment (MarynaG) [reply test/cypress/integration/reusable/open-sdk-user-data-test.js]: 🤦‍♂ fixed

**Pull request merged**